testing/internal/e2e/tests/base_plus/target_tcp_connect_postgres_test.go

```diff
@@ -76,7 +76,7 @@ func TestCliTcpTargetConnectPostgres(t *testing.T) {
 	require.NoError(t, err)
 
 	var buf bytes.Buffer
-	io.Copy(&buf, f)
+	_, _ = io.Copy(&buf, f)
```

Comment:
When I put an error check here, it fails in CI for the following reason (but it passes locally)
```
│     target_tcp_connect_postgres_test.go:80: 
│         	Error Trace:	/src/boundary/testing/internal/e2e/tests/base_plus/target_tcp_connect_postgres_test.go:80
│         	Error:      	Received unexpected error:
│         	            	read /dev/ptmx: input/output error
│         	Test:       	TestCliTcpTargetConnectPostgres
```

Not sure what the reason is yet. In CI, it does successfully pass the following assertions without the error check, so I'm inclined to keep it like this for now.